core-mysql/src/main/scala/org/apache/pekko/persistence/r2dbc/mysql/journal/MySQLJournalDao.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
```

Comment:
we need to change the source header on all the new files - see point 5 in https://github.com/apache/pekko/blob/main/CONTRIBUTING.md#pull-request-requirements

Reply:
Okay, I can revert. I do not understand the definition of "copy" and "absolutely new" in this context, so I will need some further explanation to avoid header issues in the future or just continued handholding is honestly fine.

Reply:
* if you copy/paste code from another source file (whole class or just some lines), you need to also copy the source header from the file you copied from (even if you modify that copied code)
* if you add new code to a source file and that source file does not have a source header that matches the license under which your new code is licensed, then you should add the right source header (but don't remove any existing source headers)
* you can avoid duplication
    * you might have 2 Apache source headers that might have different copyright lines - you can merge those headers into one header but with 2 copyright lines
    * in Pekko, we use 2 forms of the Apache header - one that mentions Akka but we also use the standard Apache header if none of the code in the file is derived from Akka - we don't need both Apache headers in the same file - we just use the Pekko specific one (mentioning Akka)

Reply:
Reviewing the new files on a case by case basis, it looks like some files should have the standard Apache header and some should have the special one that mentions Akka.
Maybe, we can get back to this just before we merge this. I can make the changes if the PR allows Pekko members to edit them (the default setting).

Reply:
Yeah, I was about to comment on that, because most of changes are subclass implementations without any changes that would without a doubt could be called a copy/paste.